test/DotNetWorkerTests/TestFunctionContext.cs

```diff
@@ -46,7 +66,7 @@ public TestFunctionContext(FunctionDefinition functionDefinition, FunctionInvoca
             BindingContext = new DefaultBindingContext(this);
         }
 
-        public bool IsDisposed { get; private set; }
+        public bool IsDisposed { get; internal set; }
```

Comment:
just wondering - why was this switched to internal?

Reply:
Good catch. This was a change I made in the previous iteration where I had the `IsDisposed` property being set inside the `DisposeAsync` of `TestAsyncFunctionContext`, for which I made this property's set visibility to internal. Later I decided to use the more explicit property name `IsAsyncDisposed`. Forgot to change the visibility. The latest iteration I pushed took care of this now.